sw/device/silicon_creator/manuf/base/BUILD

```diff
@@ -214,6 +214,7 @@ filegroup(
 cc_library(
     name = "ft_personalize_base",
     srcs = ["ft_personalize.c"],
+    hdrs = ["personalize_ext.h"],
```

Comment:
I think what you really want to do is make the header file its own lib like:
```
cc_library(
    name = "personalize_ext",
    hdrs = ["personalize_ext.h"],
)
```

Then include it as a dep of `ft_personalize_base` and all the extension libs.

Reply:
done